docs/store-operations/translations/index.mdx

```diff
@@ -0,0 +1,41 @@
+# Translations Admin GraphQL API (Beta)
+
+## Overview
+The Translations Admin GraphQL API is currently available on Catalyst storefronts only. This API enables merchants to translate a single website into multiple languages. 
+
+With a single storefront setup, the shopper's experience remains consistent, but the content is displayed in their preferred language. This use case is particularly common in multilingual countries like Canada and for customers selling cross-border, where the same storefront serves shoppers in multiple languages without altering the overall user experience.
+
+The API supports translations for the following components:
```

Comment:
@bc-traciporter can we add a note here, that the list will be extending ? :) 
Just to set proper expectations that it's not gonna be only categories and locations

Reply:
I changed this to the following:

The API currently supports translations for the following components, and more are expected in the future:

* Categories
* Locations